Makefile

```diff
@@ -14,4 +14,7 @@ stop:
 pitchme_to_readme:
   #Example: FOLDER_NAME=classes/01class/git/ make pitchme_to_readme
 	#Example: FOLDER_NAME=classes/01class/git/ BRANCH_NAME=MY_BRANCH make pitchme_to_readme
-	bash scripts/pitchme_to_readme.sh
\ No newline at end of file
+	bash scripts/pitchme_to_readme.sh
+
+rtr-cleanup:
+	$(ROOT_DIR)/scripts/rtr-cleanup.sh
```

Comment:
Can you check PR #366 and pass in `GH_USER` and `GH_TOKEN` the same way for script rtr-cleanup.sh?

Update README too.